UnraisinYourself/Resources/SceneDelegate.swift

```diff
@@ -20,8 +20,15 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         
         window = UIWindow(windowScene: windowScene)
         window?.makeKeyAndVisible()
-        window?.rootViewController = URTabBarController()
         
+        let welcomeViewModel = WelcomeViewController.ViewModel()
+        if welcomeViewModel.hasSeenWelcomeScreen {
```

Comment:
If user has been shown the welcome screen before, they navigate to the home screen with the tab bar. 
If user has never been shown the welcome screen, they'll see the welcome screen where they can optionally enter in their name